### ISSUE TITLE
Add mention of pinning the extension for Chrome users

### ISSUE THREAD

**Issue by Hans5958** [ScratchAddons/website-v2#49]
Hopefully self-explanatory.

A progress has been done by me on `please-pin-the-extension-chrome-users`. To whom it may concern, please do not create a PR of it in the meantime.

**Comment by Hans5958** [comment]
cc @WorldLanguages

**Comment by Hans5958** [comment]
A

**Comment by TheColaber** [comment]
It would be cool to have the BG open the settings page when installed and just show a welcome popup. I think this helps at least show users that a settings page exists.

**Comment by mxmou** [comment]
> Keep in mind that anyway, at some point we'll probably have to add the welcome screen as part of the extension itself, in order to ask for host permissions

If we do this to the transition page as well, could we remove the settings redirect instead of waiting for opportunities to move it?

**Comment by Hans5958** [comment]
# Current state (subject to change)

![image](https://user-images.githubusercontent.com/11584103/121380931-7e277500-c96f-11eb-83e8-f9ed46c976ab.png)

**Comment by Hans5958** [comment]
Yeah, I still don't have the images. Maybe soon, but someone can help me to do it.

**Comment by RedGuy12** [comment]
What's up with the dark mode icon?
________________________________

**Comment by Hans5958** [comment]
> What's up with the dark mode icon?

Don't worry about it, it's just a side effect of Firefox Screenshots.

**Comment by Hans5958** [comment]
Should I continue with this, folks?

**Comment by TheColaber** [comment]
> It would be cool to have the BG open the settings page when installed and just show a welcome popup. I think this helps at least show users that a settings page exists.

@Hans5958 I feel like what I said here would be more beneficial... basically transitioning these pages to the settings page so that the users are directly exposed to it, and if onboarding were to exist in the future, it would also happen at that moment. The pages can still stay on scratchaddons.com, but I think we shouldn't use them anymore.
